docs/octopus-rest-api/octopus-cli/allow-releaseprogression.md

```diff
@@ -73,6 +73,12 @@ Common options:
                              fatal. Defaults to 'debug'.
 ```
 
+## Basic examples
```

Comment:
```suggestion
## Basic example
```

Reply:
@wordlee on this, would you pluralize only when there is more than one (just for future reference)